docs/explanation/package-model.rst

```diff
@@ -243,14 +243,14 @@ package to the :term:`Ubuntu Archive`.
 Binary packages
 ---------------
 
-A **binary package** is a standardized format that the :term:`Package Manager`
+A **binary package** is a standardised format that the :term:`Package Manager`
 (:manpage:`dpkg(1)` or :manpage:`apt(8)`) can understand to install and
 uninstall software on a target machine. This simplifies distributing software
 to a target machine and managing the software on that machine.
 
 A Debian binary package uses the :file:`.deb` file extension and contains a set
-of files that will be installed on the host system and a set of files that
-control how the files will be (un-)installed.
+of files that are installed on the host system and a set of files that
+control how the files are installed or uninstalled.
```

Comment:
note: I think future tense is technically more correct as the files in the .deb file aren't necessarily installed (present tense).

Reply:
I should've limited the changes to spellcheck only and leave the style for later to keep it focused...

The reasononing is that the future tense is generally discouraged in docs (in this case, the meaning is implied to be "files that are installed during the installation process"), but I understand your concern. I'll reverse this and leave it for later.